src/components/AboutPage/AboutSection/AboutSection.js

```diff
@@ -3,18 +3,17 @@ import { Container, Row, Col } from 'react-bootstrap';
 import townImg from './imgs/undraw_quite_town_mg2q1.svg';
```

Comment:
This file should not be changed. accept the `useTranslation` changes

fix here and all